main/php.h

```diff
@@ -179,6 +179,10 @@ END_EXTERN_C()
 #define explicit_bzero php_explicit_bzero
 #endif
 
+BEGIN_EXTERN_C()
+PHPAPI int php_safe_bcmp(const zend_string *a, const zend_string *b);
```

Comment:
*maybe* add a comment either here to indicate that ``-1`` is if the strings do not have the same lengths, ``0`` if they match and ``1`` if they are not equal?